Sources/ArcGISToolkit/Components/FloatingPanel/FloatingPanel.swift

```diff
@@ -272,21 +274,3 @@ private struct RoundedCorners: Shape {
         return Path(path.cgPath)
     }
 }
-
-private extension View {
-    /// Clips this view to its bounding frame, with the specified corner radius, on the specified corners.
-    /// - Parameters:
-    ///   - radius: The radius used to round the corners.
-    ///   - corners: The corners to be rounded.
-    /// - Returns: A view that clips this view to its bounding frame with the specified corner radius and
-    /// corners.
-    func cornerRadius(
-        _ radius: CGFloat,
-        corners: UIRectCorner
-    ) -> some View {
-        clipShape(RoundedCorners(
-            corners: corners,
-            radius: radius
-        ))
-    }
```

Comment:
This function wasn't carrying its own weight in my opinion so I dropped it and now call `clipShape` directly.